src/runtime/store/index.ts

```diff
@@ -66,46 +66,37 @@ export function writable<T>(value: T, start: StartStopNotifier<T> = noop): Writa
 	const subscribers: Array<SubscribeInvalidateTuple<T>> = [];
 
 	function set(new_value: T): void {
-		if (safe_not_equal(value, new_value)) {
-			value = new_value;
-			if (stop) { // store is ready
-				const run_queue = !subscriber_queue.length;
-				for (let i = 0; i < subscribers.length; i += 1) {
-					const s = subscribers[i];
-					s[1]();
-					subscriber_queue.push(s, value);
-				}
-				if (run_queue) {
-					for (let i = 0; i < subscriber_queue.length; i += 2) {
-						subscriber_queue[i][0](subscriber_queue[i + 1]);
-					}
-					subscriber_queue.length = 0;
-				}
-			}
+		if (!safe_not_equal(value, new_value)) return;
+		value = new_value;
+		if (!stop) return;
```

Comment:
As much as I'm absolutely 100% against comments, it's not my call here, and we've lost a comment. I think it should be either reinstated, or better:

```js
const store_is_ready = stop
```

In a perfect world:

```js
const store_is_ready = stop
if (!store_is_ready) { return }
```

However I realise this is runtime code, so I'm guessing there is an emphasis on bytes down the wire + terseness over readability. I don't know how TS optimises all of this stuff, so therefore maybe the comment is the perfect solution here. Ideally though, I'd aim for readability of the code.